lib/file_transfer_component/commands/initiate.rb

```diff
@@ -2,12 +2,12 @@ module FileTransferComponent
   module Commands
     class Initiate
       include Command
+      attr_writer :file_id
 
-      initializer :file_id, :name, :uri
+      initializer :name, :uri
```

Comment:
Seems like  file_id should be an initializer parameter. Otherwise, it can be left blank. ie: right now it's optional for this object to exist without a file ID.